tests/models/test_amp.py

```diff
@@ -108,7 +108,7 @@ def test_amp_cpus(tmpdir, accelerator, precision, num_processes):
     "accelerator",
     [
         None,
-        pytest.param("dp", marks=pytest.mark.skip("dp + amp not supported currently")),  # TODO
+        "dp",
```

Comment:
On my system (two A10), this test passed for me (both 1 and 2 GPUs) with bfloat16.